docs/source/using/addons.adoc

```diff
@@ -161,6 +161,21 @@ $ minishift addons enable anyuid
 $ minishift addons disable anyuid
 ----
 
+[[apply-addons]]
+== Applying add-ons
+Enabled add-ons automatically get executed during link:../command-ref/minishift_start{outfilesuffix}[`minishift start`].
+
+Also add-ons can be explictly executed with link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
+
+The following examples show how to execute the _anyuid_ and _admin-user_ add-ons.
```

Comment:
The following example shows how to explicitly apply the _anyuid_ and _admin-user_ add-ons.